buildbot-buildmaster/buildbot/multiarch_dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:bionic
+
+# hints related to apt and multiple architectures:
+# https://stackoverflow.com/questions/37706635/in-docker-apt-get-install-fails-with-failed-to-fetch-http-archive-ubuntu-com
+# https://wiki.ubuntuusers.de/sources.list/
+# https://wiki.debian.org/QemuUserEmulation
+# https://wiki.debian.org/Multiarch/HOWTO
+
+RUN mv /etc/apt/sources.list /etc/apt/sources.list.old
+
+RUN printf \
+"deb [arch=amd64,i386] http://de.archive.ubuntu.com/ubuntu bionic main restricted universe multiverse\n\
+deb [arch=amd64,i386] http://de.archive.ubuntu.com/ubuntu bionic-updates main restricted universe multiverse\n\
+deb [arch=amd64,i386] http://de.archive.ubuntu.com/ubuntu bionic-security main restricted universe multiverse\n\
+deb [arch=amd64,i386] http://de.archive.ubuntu.com/ubuntu bionic-backports main restricted universe multiverse\n\
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ bionic main universe restricted multiverse\n\
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ bionic-security main universe restricted multiverse\n\
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ bionic-updates main universe restricted multiverse\n\
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ bionic-backports main universe restricted multiverse\n"\
+> /etc/apt/sources.list
+
+
+RUN apt-get update && apt-get install -y git
```

Comment:
All `apt` commands, including the update must be combined in one line via `&&`-style command concatenation. Otherwise, cached versions of the `update` step may interfer with subsequent installation steps that try to access outdated repositories

Reply:
Fixed.

Reply:
Fixed.